Upstatement/craft-starter
# Smooth out install

# Changes

- Simplifies and fixes the initial MySQL dumpfile loading by the `mysql` container. (Fixes #2).
- Removes an install step by a) adding the Docker DB container details to `.env.example` and b) copying `.env.example` -> `.env` during `./bin/install`
- Uses the Craft CLI to generate a secure key during `./bin/install`
- Removes some transient files from the repo and adds them to `.gitignore` (`storage/logs` and `public/static/dist`)
- Adds `config/schema.yml` to `.prettierignore` and re-exports it

## How To Test

1. Follow the install steps from the `README` in a fresh clone of the repo with this branch
2. Confirm that there are no hiccups!
